crypto/client.go

```diff
@@ -15,8 +16,16 @@ func New(config Config) (Client, error) {
 		if config.Type != "" {
 			fmt.Println("Invalid encryption type, using default")
 		}
-		return NewAESOpenPGPClient(config)
-	case "aes-openpgp":
-		return NewAESOpenPGPClient(config)
+		fallthrough
+	case "aes_openpgp":
+		// Remove other settings
+		// TODO(leon): This is shitty.
+		config.Settings.AESGCMSettings = nil
```

Comment:
What's going on here? It looks like this is set to `DefaultAESGCMSettings()` when the config is loaded.